public/app/features/plugins/sandbox/sandbox_plugin_loader_registry.ts

```diff
@@ -62,21 +64,34 @@ export async function isPluginFrontendSandboxEligible({
     return false;
   }
 
-  // don't run grafana-signed plugins in sandbox
+  // skip cache for jest tests
+  if (!process.env.JEST_WORKER_ID && pluginElegibilityCache.has(pluginId)) {
```

Comment:
this is the most reliable method I found to skip the cache inside jest (otherwise unit tests will fail because of it)